build.gradle

```
@@ -1,7 +1,9 @@
 buildscript {
     repositories {
         gradlePluginPortal()
```

I think we may want the same thing here, the gradle plugin portal redirects to other repositories (thus the pain when jcenter goes down)
```suggestion
        gradlePluginPortal() {
            metadataSources { mavenPom(); ignoreGradleMetadataRedirection() }
        }
```

---

Probably worth excavating this out to our oss projects